mozilla/blurts-server
# MNTOR-2120 - re enable breach alerts with pubsub

# References: 
Jira: MNTOR-2120




# Description

Re-enable email breach alerts using GCP PubSub for queue, and cron job for processing results (to be run hourly).

See https://github.com/mozilla/blurts-server/pull/2976 for rationale behind this approach.

# How to test

See README changes for running local emulator, and set env vars:
```
GCP_PUBSUB_PROJECT_ID
GCP_PUBSUB_TOPIC_NAME
GCP_PUBSUB_SUBSCRIPTION_NAME
```

# Checklist (Definition of Done)
